WalletWasabi.Documentation/WasabiSetupRegtest.md

```diff
@@ -2,29 +2,30 @@
 
 ## Why do this?
 
-RegTest is a local testing environment in which developers can almost instantly generate blocks on demand for testing events, and can create private satoshis with no real-world value. Running Wasabi Backend on RegTest allows you to emulate network events and observe how the Backend and the Client react on that.
+RegTest is a local testing environment in which developers can almost instantly generate blocks on demand for testing events, and create private satoshis with no real-world value. Running Wasabi Backend on RegTest allows you to emulate network events and observe how the Backend and the Client react on that.
 You do not need to download the blockchain for this setup!
 
-## Setup Bitcoin core with RegTest
+## Setup Bitcoin Core with RegTest
 
-Follow [this guide](https://bitcoin.org/en/developer-examples#testing-applications).
+Follow [this guide](https://bitcoin.org/en/developer-examples#regtest-mode).
 
 Todo:
-1. Install [Bitcoin core](https://bitcoin.org/en/bitcoin-core/) on your computer.
-2. Start Bitcoin core with: bitcoin-qt.exe -regtest then quit immediately - in this way the data directory and the config files will be generated. 
+
+1. Install [Bitcoin Core](https://bitcoin.org/en/bitcoin-core/) on your computer.
+2. Start Bitcoin Core with: bitcoin-qt.exe -regtest then quit immediately. In this way the data directory and the config files will be generated.
 ```
 Windows: "C:\Program Files\Bitcoin\bitcoin-qt.exe" -regtest
-OSX: "/Applications/Bitcoin-Qt.app/Contents/MacOS/Bitcoin-Qt" -regtest
+macOS: "/Applications/Bitcoin-Qt.app/Contents/MacOS/Bitcoin-Qt" -regtest
 Linux:
 ```
-2. Go to Bitcoin core settings directory. If the directory is missing run core bitcoin-qt, then quit immediately - in this way the data directory and the config files will be generated. 
+3. Go to Bitcoin Core settings directory. If the directory is missing run core bitcoin-qt, then quit immediately. In this way the data directory and the config files will be generated.
```

Comment:
I got confused by what is the `settings directory`, but it is what is mentioned in lines 23-25, `/.bitcoin/`?